### ISSUE TITLE
/discord reload not firing

### ISSUE THREAD

**Issue by BigJazzz** [nguyenquyhy/DiscordBridge#49]
Hey guys

Trying to do `/discord reload` after changing the config.json (added role prefixes), however I'm not getting any feedback in chat to say it's been reloaded, and when I run test messages from Discord it's still the plain grey name with no prefix.

**Comment by BigJazzz** [comment]
Please ignore. For anyone else having this issue, make sure you have comma'd correctly. :D

**Comment by BigJazzz** [comment]
Hey guys

So the issue I was having has been sorted, but I'm noticing that running `/discord reload` doesn't reload properly. Changes to the chat templates and bot tokens require a complete server restart before the changes take effect.

**Comment by nguyenquyhy** [comment]
Any log?

**Comment by BigJazzz** [comment]
I'll try to grab one.

**Comment by BigJazzz** [comment]
The only lines in the log I can seem to see are these ones. I'll ask the other admin team members and see if they can find anything more substantial: https://ssby.me/ss/272916170229.png

**Comment by nguyenquyhy** [comment]
You are right. The config is cached incorrectly 😄 . This will be fixed in the next update.
